<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1009</width>
    <height>682</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="mainLayout">
      <item>
       <layout class="QVBoxLayout" name="propertiesLayout">
        <item>
         <widget class="QGroupBox" name="penPropertiesGBox">
          <property name="title">
           <string>Pen Properties</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QHBoxLayout" name="penColorHLayout">
             <item>
              <widget class="QLabel" name="penColorLabel">
               <property name="text">
                <string>Pen Color</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="penColorButton">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(0, 0, 0);</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
               <property name="autoExclusive">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="colorsHLayout">
             <item>
              <widget class="QPushButton" name="colorButton_1">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(85, 0, 127);</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="autoExclusive">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="colorButton_2">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(50, 14, 127);</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="autoExclusive">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="colorButton_3">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(0, 0, 255);</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="autoExclusive">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="colorButton_4">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(0, 255, 0);</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="autoExclusive">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="colorButton_5">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 0);</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="autoExclusive">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="colorButton_6">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 85, 0);</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="autoExclusive">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="colorButton_7">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 0, 0);</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="autoExclusive">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="penWidthHLayout">
             <item>
              <widget class="QLabel" name="penWidthLabel">
               <property name="text">
                <string>Pen Width</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="penWidthSpinBox">
               <property name="minimum">
                <number>2</number>
               </property>
               <property name="maximum">
                <number>12</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="penStyleHLayout">
             <item>
              <widget class="QLabel" name="penStyleLabel">
               <property name="text">
                <string>Pen Style</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="penStyleComboBox">
               <item>
                <property name="text">
                 <string>Solid</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Dashed</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Dotted</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Dot Dashed</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="shapesPropertiesGBox">
          <property name="title">
           <string>Shapes Properties</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <layout class="QHBoxLayout" name="borderWidthHLayout">
             <item>
              <widget class="QLabel" name="borderWidthLabel">
               <property name="text">
                <string>Border Width</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="borderWidthSpinBox">
               <property name="minimum">
                <number>2</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="borderStyleHLayout">
             <item>
              <widget class="QLabel" name="borderStyleLabel">
               <property name="text">
                <string>Border Style</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="borderStyleComboBox"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="borderColorHLayout">
             <item>
              <widget class="QLabel" name="borderColorLabel">
               <property name="text">
                <string>Border Color</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="borderColorButton">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="fillStyleHLayout">
             <item>
              <widget class="QLabel" name="fillStyleLabel">
               <property name="text">
                <string>Fill Style</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="fillStyleComboBox"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="fillColorHLayout">
             <item>
              <widget class="QLabel" name="fillColorLabel">
               <property name="text">
                <string>Fill Color</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="fillColorButton">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="canvasPropertiesGBox">
          <property name="title">
           <string>Canvas Properties</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <layout class="QVBoxLayout" name="showGridVLayout">
             <item>
              <widget class="QCheckBox" name="showGridCheckBox">
               <property name="text">
                <string>Show Grid</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="centerSceneButton">
               <property name="text">
                <string>Center Scene</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="sceneBackgroundHLayout">
             <item>
              <widget class="QLabel" name="sceneBackgroundLabel">
               <property name="text">
                <string>Scene Background :</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="sceneBackgroundButton">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="selectShapeGBox">
          <property name="title">
           <string>Select Shape</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <widget class="QLabel" name="filledEllipseLabel">
             <property name="styleSheet">
              <string notr="true">image: url(:/images/images/ellipse.png);</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="qtQuickLabel">
             <property name="styleSheet">
              <string notr="true">image: url(:/images/images/quick.png);</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="filledRectangleLabel">
             <property name="styleSheet">
              <string notr="true">image: url(:/images/images/rectangle.png);</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="filledStarLabel">
             <property name="styleSheet">
              <string notr="true">image: url(:/images/images/star.png);</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGraphicsView" name="graphicsView"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1009</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAdd_image"/>
    <addaction name="actionQuit"/>
    <addaction name="actionSave"/>
    <addaction name="actionLoad"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="actionCut"/>
    <addaction name="actionPaste"/>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionCursor"/>
    <addaction name="actionPen"/>
    <addaction name="actionEraser"/>
    <addaction name="actionEllipse"/>
    <addaction name="actionRectangle"/>
    <addaction name="actionStar"/>
   </widget>
   <widget class="QMenu" name="menuPreferences">
    <property name="title">
     <string>Preferences</string>
    </property>
    <addaction name="actionLanguage_Settings"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_this_app"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTools"/>
   <addaction name="menuPreferences"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSave"/>
   <addaction name="actionLoad"/>
   <addaction name="separator"/>
   <addaction name="actionCopy"/>
   <addaction name="actionCut"/>
   <addaction name="actionPaste"/>
   <addaction name="separator"/>
   <addaction name="actionCursor"/>
   <addaction name="actionPen"/>
   <addaction name="actionEraser"/>
   <addaction name="actionEllipse"/>
   <addaction name="actionStar"/>
   <addaction name="actionRectangle"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
  </widget>
  <action name="actionAdd_image">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/open.png</normaloff>:/images/images/open.png</iconset>
   </property>
   <property name="text">
    <string>Add Image</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/close.png</normaloff>:/images/images/close.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/save.png</normaloff>:/images/images/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/open.png</normaloff>:/images/images/open.png</iconset>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/copyIcon.png</normaloff>:/images/images/copyIcon.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/cutIcon.png</normaloff>:/images/images/cutIcon.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/pasteIcon.png</normaloff>:/images/images/pasteIcon.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/undoIcon.png</normaloff>:/images/images/undoIcon.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/redoIcon.png</normaloff>:/images/images/redoIcon.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionCursor">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/cursor.png</normaloff>:/images/images/cursor.png</iconset>
   </property>
   <property name="text">
    <string>Cursor</string>
   </property>
  </action>
  <action name="actionPen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/pen.png</normaloff>:/images/images/pen.png</iconset>
   </property>
   <property name="text">
    <string>Pen</string>
   </property>
  </action>
  <action name="actionEraser">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/eraser.png</normaloff>:/images/images/eraser.png</iconset>
   </property>
   <property name="text">
    <string>Eraser</string>
   </property>
  </action>
  <action name="actionEllipse">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/circle.png</normaloff>:/images/images/circle.png</iconset>
   </property>
   <property name="text">
    <string>Ellipse</string>
   </property>
  </action>
  <action name="actionRectangle">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/rectangle1.png</normaloff>:/images/images/rectangle1.png</iconset>
   </property>
   <property name="text">
    <string>Rectangle</string>
   </property>
  </action>
  <action name="actionStar">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/star.png</normaloff>:/images/images/star.png</iconset>
   </property>
   <property name="text">
    <string>Star</string>
   </property>
  </action>
  <action name="actionLanguage_Settings">
   <property name="text">
    <string>Language Settings</string>
   </property>
  </action>
  <action name="actionAbout_this_app">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/images/about.png</normaloff>:/images/images/about.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
